tests/unit/C/plugins/common/test_omf_translation.cpp

```diff
@@ -17,6 +17,8 @@
 using namespace std;
 using namespace rapidjson;
 
+#define TYPE_ID "1234"
```

Comment:
Eventually we want to be able to have type id changed, but I am assuming this is a temporary mechanism

Reply:
Only for tests